client/signup/jetpack-connect/authorize-form.jsx

```diff
@@ -159,6 +161,10 @@ const LoggedInForm = React.createClass( {
 		const { autoAuthorize, queryObject } = this.props.jetpackConnectAuthorize;
 		debug( 'Checking for auto-auth on mount', autoAuthorize );
 		this.props.recordTracksEvent( 'jpc_auth_view' );
+		if ( ! this.props.isCalypsoStartedConnection ) {
+			debug( 'refetching sites on mount' );
```

Comment:
I added in this debug. I've noticed that we are requesting sites even on connections that start in Calypso. `isCalypsoStartedConnection` seems to be broken.